components/tabs/__tests__/index.test.tsx

```diff
@@ -50,73 +42,169 @@ describe('Tabs', () => {
     expect(value).toBe('2')
   })
 
-  it('should ignore events when tab disabled', () => {
-    const changeHandler = jest.fn()
+  it('should change style when hover header', () => {
     const wrapper = mount(
-      <Tabs initialValue="1" onChange={changeHandler}>
+      <Tabs initialValue="1">
         <Tabs.Item label="label1" value="1">
           1
         </Tabs.Item>
-        <Tabs.Item label="label2" value="2" disabled>
+        <Tabs.Item label="label2" value="2">
           2
         </Tabs.Item>
       </Tabs>,
     )
+    wrapper.find('header').find('.nav').at(1).simulate('mouseEnter', nativeEvent)
+    expect(wrapper.find('header').find('.label').at(1).get(0).props.style).toHaveProperty(
+      'color',
+      '#0054fe',
```

Comment:
Why not using snapshot here? Use hex color will easily break the test if there're style changes.

Reply:
Or you can just check the existence of classname